install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1704,10 +1704,12 @@ dashboards:
 
 # -- Configure Istio proxy options.
 proxy:
-  # Deprecated: use envoy.prometheus instead
+
   prometheus:
+    # -- Deprecated in favor of envoy.prometheus.enabled
     enabled: true
-    port: "9964"
```

Comment:
Needs upgrade note as well.

Reply:
@aanm 

thanks. added the deprecation of `proxy.prometheus.enabled` & `proxy.prometheus.port` in favor of their counterparts in `envoy.prometheus.*` to the upgrade guide.